cdxev/vex.py

```diff
@@ -0,0 +1,180 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+from typing import Any, Union
+
+
+def init_vex_header(input_file: dict[str, Any]) -> dict[str, Any]:
+    """
+    Copy important keys and values from input_file to output_file
+
+    Parameters
+    ----------
+    input_file: dict
+        A VEX/SBOM dictionary from which the information should be copied
+
+    Returns
+    -------
+    output_file: dict
+        A dict with important header pairs
+    """
+
+    output_file = {}
+    output_file["bomFormat"] = input_file.get("bomFormat", "")
+    output_file["specVersion"] = input_file.get("specVersion", "")
+    output_file["version"] = input_file.get("version", 0)
+    return output_file
+
+
+def get_list_of_ids(input_file: dict[str, Any], scheme: str) -> str:
+    """
+    Get a list of vulnerability IDs.
+
+    Parameters
+    ----------
+    input_file: dict
+        A dictionary to search for values
+
+    Returns
+    -------
+    list_str: str
+        A string with all IDs in .csv format
+    """
+
+    list_str = ""
+    if scheme == "default":
+        list_str += "ID,RefID,Description,Status\n"
+        for vulnerability in input_file.get("vulnerabilities", []):
+            vul_id = vulnerability.get("id", "-")
+            vul_ref_id = vulnerability.get("references", [{"id": "-"}])[0].get(
+                "id", "-"
+            )
+            vul_description = vulnerability.get("description", "-")
```

Comment:
you are not sanitizing the input, so if e.g. the description has "\n" in it, it introduces this within your output, e.g.

			"description": "Microsoft is releasing this security advisory to provide information about a vulnerability in .NET 6.0, .NET 5.0 and .NET Core 3.1. This advisory also provides guidance on what developers can do to update their applications to remove this vulnerability.\n\nA vulnerability exists in .NET 6.0, .NET 5.0 and .NET core 3.1 where a malicious client can manipulate cookies and cause a Denial of Service.\n\n###Affected Software

Results in:
![image](https://github.com/user-attachments/assets/6da605f7-d686-4db5-933f-27dc1ad7a1b8)